package.json

```diff
@@ -18,7 +18,7 @@
     "lodash": "^3.6.0",
     "mime": "^1.3.4",
     "napa": "^1.2.0",
-    "protagonist": "^0.19.0",
+    "protagonist": "^1.3.3",
```

Comment:
Let's also remove the Aglio dependency as part of this PR? Ballad doesn't directly need it